chromadb/test/test_api.py

```diff
@@ -706,15 +706,15 @@ def test_metadata_update_get_int_float(client):
 def test_metadata_validation_add(client):
     client.reset()
     collection = client.create_collection("test_metadata_validation")
-    with pytest.raises(ValueError, match="metadata"):
+    with pytest.raises(Exception, match="metadata"):
```

Comment:
I don't understand the need for this change and would rather we didn't change this contract